draft-ietf-quic-qpack.md

```diff
@@ -718,15 +718,15 @@ safe to process the rest of the block.  If Largest Reference is greater than
 zero, the encoder transforms it as follows before encoding:
 
 ~~~
-   LargestReference = LargestReference mod 2*MaxEntries + 1
+   LargestReference = LargestReference mod (2*MaxEntries) + 1
```

Comment:
Do you mean `LR % ((2*ME)+1)` or `(LR % (2*ME))+1`?  I think the latter, but I think that it's probably best to double-down on the parentheses here.

Reply:
I mean the latter.  I can add more parentheses.